ros2bag/ros2bag/verb/play.py

```diff
@@ -85,6 +85,9 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '-d', '--delay', type=positive_float, default=0.0,
             help='Sleep duration before play (each loop), in seconds. Negative durations invalid.')
+        parser.add_argument(
+            '--disable-keyboard-controls', action='store_true',
+            help='disables keyboard controls for pause/resume/play-next')
```

Comment:
might remove the `pause/resume/play-next` wording because i think we will add even more controls and i'd rather not enumerate them all here (or have to remember to update it)